docs/data-model.md

```diff
@@ -298,11 +298,20 @@ required for a valid set of mutations.
 In simulations, trees can be thought of as spread across space, and it is
 helpful for inferring demographic history to record this history.
 Migrations are performed by individual ancestors, but most likely not by an
-individual whose genome is tracked as a `node` (as in a discrete-deme model they are
-unlikely to be both a migrant and a most recent common ancestor).  So,
-`tskit` records when a segment of ancestry has moved between
-populations. This table is not required, even if different nodes come from
-different populations.
+individual whose genome is tracked as a `tskit` node (as in a discrete-deme
+model they are unlikely to be both a migrant and a most recent common ancestor).
```

Comment:
this is a bit confusing to me-- i get what you mean, but had to read it twice-- guessing that someone with less experience than me would also be confused. i wonder if there is a way that this could be rephrased?

Maybe "While individuals migrate during the simulations, the ancestral chunks of those individuals is unlikely to be captured as a tskit node, because it would be the rare case in which a particular chunk would be both a migrant and an common ancestor"